extensions/fluent/src/org/exist/fluent/Database.java

```diff
@@ -104,20 +104,17 @@ static void configureRootCollection(Path configFile) {
 		}
 		
 		// Now force reload and reindex so it'll pick up the new settings.
-		Transaction tx = db.requireTransactionWithBroker();
 		try {
-			pool.getConfigurationManager().addConfiguration(tx.tx, tx.broker, tx.broker.getCollection(XmldbURI.ROOT_COLLECTION_URI), configXml.toString());
-			tx.commit();
-			DBBroker broker = db.acquireBroker();
-			try {
-				broker.reindexCollection(XmldbURI.ROOT_COLLECTION_URI);
-			} finally {
-				db.releaseBroker(broker);
-			}
+            try(final Transaction tx = db.requireTransactionWithBroker()) {
```

Comment:
indenting is not OK here? [readability]

Reply:
It was a tabs vs spaces thing. I have fixed it.